src/application/components/Explorer/Explorer.tsx

```diff
@@ -174,64 +186,65 @@ export const Explorer = ({ navigationProps, embeddedExplorerProps }: {
       });
 
       observer.subscribe((response: QueryResult) => {
-        if(response.networkStatus === NetworkStatus.error) {
-          embeddedExplorerIFrame.contentWindow?.postMessage({
-            name: SCHEMA_ERROR,
-            schema: response.data,
```

Comment:
schema isn't necessary to send here - there is no schema - we don't expect this on the studio-ui side